android/src/main/java/com/mapbox/mapboxgl/MapboxMapController.java

```diff
@@ -460,6 +460,21 @@ public void onMethodCall(MethodCall call, MethodChannel.Result result) {
         result.success(reply);
         break;
       }
+      case "map#toScreenLocationBatch": {
+        List<List<Object>> reply = new ArrayList<List<Object>>();
+
+        for (List<Object> latLng : (List<List<Object>>)call.argument("param")) {
+          PointF pointf = mapboxMap.getProjection().toScreenLocation(new LatLng((Double)latLng.get(0), (Double)latLng.get(1)));
+
+          List<Object> point = new ArrayList();
+          point.add(pointf.x);
+          point.add(pointf.y);
```

Comment:
Thinking about it more, the best approach would be to just return a single flattened list of `float`s (of 2x length) and combine them on the dart side.

At the very least it would be much more efficient to return a `List<float[]>`, as using collections can be expensive.

Reply:
I changed the structure.